Gemfile.lock

```diff
@@ -113,29 +111,12 @@ GEM
       erubi (>= 1.0.0)
       rack (>= 0.9.0)
     builder (3.2.4)
-    chunky_png (1.4.0)
     cocoon (1.2.15)
     coderay (1.1.2)
-    compass (1.0.3)
-      chunky_png (~> 1.2)
-      compass-core (~> 1.0.2)
-      compass-import-once (~> 1.0.5)
-      rb-fsevent (>= 0.9.3)
-      rb-inotify (>= 0.9)
-      sass (>= 3.3.13, < 3.5)
-    compass-core (1.0.3)
-      multi_json (~> 1.0)
-      sass (>= 3.3.0, < 3.5)
-    compass-import-once (1.0.5)
-      sass (>= 3.2, < 3.5)
-    compass-rails (2.0.5)
-      compass (~> 1.0.0)
-      sass-rails (< 5.1)
-      sprockets (< 2.13)
     concurrent-ruby (1.1.6)
-    connection_pool (2.2.3)
-    countries (3.0.1)
-      i18n_data (~> 0.10.0)
+    connection_pool (2.2.5)
+    countries (3.1.0)
+      i18n_data (~> 0.11.0)
```

Comment:
Should we be updating gem versions for gems outside the scope? 🤷 

We'll be updating them anyway, so probably OK.